scripts/build/gen_isr_tables.py

```diff
@@ -278,16 +276,18 @@ def bit_mask(bits):
     return mask
 
 def update_masks():
-    global FIRST_LVL_INTERRUPTS
```

Comment:
`global` doesn't seem necessary to modify a global list

Reply:
It's only needed to ensure that a local variable isn't instantiated instead of using the global one.

Likely without this, the only reason it works is because the variable is read before it's written.

Probably safe to keep though.

Reply:
If they can just be local, it would be preferable.